crates/kitsune2/tests/integration.rs

```diff
@@ -187,12 +183,12 @@ async fn two_node_gossip() {
         {
             break;
         } else {
-            println!(
+            tracing::info!(
```

Comment:
Use tracing rather than printing so I can see the progress it's making if this fails on CI.